pkg/logs/input/file/scanner.go

```diff
@@ -185,23 +186,50 @@ func (s *Scanner) launchTailers(source *config.LogSource) {
 		if _, isTailed := s.tailers[file.Path]; isTailed {
 			continue
 		}
-		var tailFromBeginning bool
+
+		mode, found := TailingModeFromString(source.Config.TailingMode)
+		if !found {
+			log.Infof("Invalid or no tailing mode provided, tailing from the end of the file %v", file.Path)
```

Comment:
Do we really need this log ?

Reply:
Probably not, I think I added it while working on the feature, will remove it or move it to debug/trace level.